docs/DeveloperGuide.md

```diff
@@ -2,32 +2,143 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+- NUS SC2113 Duke project
+- [NUS SC2113 tp](https://github.com/nus-cs2113-AY2223S2)
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+All commands will be fetched by the getUserCommand() method in the TextUi class. The parseCommand() method in the Parser class will then parse the command and pass it to the respective command classes. The command classes will be responsible for executing the relevant features.
+
+### [Proposed] Search book by title
+
+![](images/Search%20class%20diagram.png)
+Description: Searches for a book using the title  
+Format: search -title TITLE
+Example:
+- search -title Python Programming  
+For successful searches, the program will output the relevant book along with the book details: ISBN, Title, Author, Topic. For unsuccessful searches, the program will output a string informing the user that there is no match with the input title from the inventory.
```

Comment:
You should leave one more line between line 16 and 17 or specify a line break `<br />`. Cos currently it looks like this:
![image](https://user-images.githubusercontent.com/88535855/228737029-2539d9b1-515e-4862-9076-9227123c2f71.png)

As you can see, the Format and Example is on the same line, which doesn't seem to be intentional.